Prog/clean-montgomery.Rmd

```diff
@@ -683,6 +692,8 @@ rm(x, y)
 There are a number of properties that have numbers or symbols in their house numbers that prevent a collapse. This next step removes those characters so the collapse is clean.
 This will also recategorize properties that have a missing house number as properties that have a missing address. This is important for the collapse because it could cause us to combine two observations that would otherwise be seperate.
 
+Overwrite jur and remove intermediate data frames
```

Comment:
i don't want you to overwrite jur...i'm fine if you remove some of the intermediate steps occasionally but the point of renaming them is so one can compare versions.